src/Tools/MicroBuild/Build.proj

```diff
@@ -25,6 +25,8 @@
 
     <MSBuild Projects="$(ProjectDir)Roslyn.sln" BuildInParallel="true" Properties="DeployExtension=false" />
 
+    <Exec Command="$(NuGetPackageRoot)\GitLink\$(GitLinkVersion)\lib\net45\GitLink.exe $(ProjectDir) -u https://github.com/dotnet/roslyn -f Roslyn.sln -c $(Configuration)" />
```

Comment:
@jaredpar It depends per source server, but in GitHub it's hard to find the actual raw files on the server (since it's a merge of both original repository + fork). I've been doing some testing, but couldn't find a reliable way to find the actual raw files for a PR. 

If you have a reliable way to specify the url (because I don't think the specified url in this script will point to the *right* PR-files), I would love to hear about that.

I do understand why you want it for PR's (although it's a very rare use-case for most people), but it makes total sense.